core/src/services/dropbox/core.rs

```diff
@@ -82,6 +89,9 @@ impl DropboxCore {
         }
         if let Some(mime) = content_type {
             request_builder = request_builder.header(header::CONTENT_TYPE, mime);
+        } else {
```

Comment:
We can use `request_builder = request_builder.header(header::CONTENT_TYPE, mime.unwrap_or("xxxx"));` instead.